### DISCUSSION TITLE
MPR in grid_stat

### DISCUSSION THREAD

**Discussion by AnastasiaBundel** [dtcenter/METplus#991] [Configuration] [answered]
Hello! Is there an output_flag option MPR = BOTH in grid_stat configuration file similar to the point_stat configuration file? I added one, but it didn't produce any output.

**Comment by j-opatz** [comment, marked as answer]
Hello and thank you for your question. The Matched Pairs (MPR) line type output is reserved for comparisons between point data and gridded data, like using PointStat. Take a look at the MET User's Guide entry for the GridStat Configuration [file](https://met.readthedocs.io/en/develop/Users_Guide/grid-stat.html#grid-stat-configuration-file). As listed in the output_flag section, MPR does not appear in the available list of output line types.
If you have any additional questions on what line types are available for the various tools, you can usually find them in the "configuration file" subsection of the "Practical Information" sections in each tool's chapter.

**Reply by AnastasiaBundel** [reply]
Many thanks for the answer!
Yes, I saw that the default config file, as well as an example in the manual, doesn't contain MPR option. But I wanted to be sure, as it seems that the logic for the calculation of most of the traditional scores in GridStat is essentially the same as in PointStat. Moreover, the matched pairs could be saved in a netcdf file. So I wondered. Such an option for line type would make easier the aggrefation of some of the GridStat scores in StatAnalysis, i think.
